airbyte-webapp/src/components/ui/PillSelect/index.stories.tsx

```diff
@@ -48,3 +53,32 @@ Multi.args = {
   isMulti: true,
   value: ["first_name", "last_name"],
 };
+
+const optionsWithTwoValues = [
+  {
+    value: "test1",
+    label: "dog | cat",
+    pillLabel: ["dog", "cat"],
```

Comment:
The pillLabel value is. not necessary. We can assume that if the label is an array, then when it needs to be displayed as text, it should be joined with the `" | "` string

Reply:
I decided to put additional `pillLabel` property since the original `label` is used internally by the `Popout` component and the array `label` breaks the `Popout` component rendering
![image](https://user-images.githubusercontent.com/25852567/205361273-cd061427-e197-43bd-8515-36dda18cd258.png)
but if you think that it is ok to fix the `Poput` component so it can receive `label` as array in this PR I can do it

Reply:
Yes, that work should be done by the component